lte/gateway/c/core/oai/tasks/nas/nas_proc.cpp

```diff
@@ -49,7 +49,7 @@ extern "C" {
 #endif
 
```

Comment:
can you move 
https://github.com/pruthvihebbani/magma/blob/cf4c51e84a4255b83635b3ce8046ed439875f486/lte/gateway/c/core/oai/tasks/nas/nas_proc.cpp#L66
to the top